deploy/cloudbuild-release.yaml

```diff
@@ -1,13 +1,18 @@
+# using default substitutions, provided by Google Container Builder
+# see: https://cloud.google.com/container-builder/docs/configuring-builds/substitute-variable-values#using_default_substitutions
 steps:
 # Build the container that will do the go build.
   - name: 'gcr.io/cloud-builders/docker'
     args: ['build', '-t', 'gcr.io/k8s-skaffold/skaffold:$TAG_NAME', '-f', 'deploy/skaffold/Dockerfile', '.']
 # Do the go build.
   - name: 'gcr.io/k8s-skaffold/skaffold:$TAG_NAME'
-    args: ['make', 'cross']
+    args: ['make', 'cross', 'docs']
```

Comment:
this fails @balopat 
docker is not in the integration test image. I would move this to a separate step (I don't want to add asciidoctor to the integration/published skaffold image

Reply:
sounds good. I'll fix it up.